tools/ci/pr_preview.py

```diff
@@ -292,139 +276,146 @@ def get_pull_requests_with_open_ref(self):
 def is_open(pull_request):
     return not pull_request['closed_at']
 
+
 def has_mirroring_label(pull_request):
     for label in pull_request['labels']:
         if label['name'] == LABEL:
             return True
 
     return False
 
+
 def should_be_mirrored(project, pull_request):
     return (
         is_open(pull_request) and (
             has_mirroring_label(pull_request) or (
                 pull_request['user']['login'] not in AUTOMATION_GITHUB_USERS and
                 pull_request['author_association'] in TRUSTED_AUTHOR_ASSOCIATIONS
             )
-        ) and
-        # Query this last as it requires another API call to verify
-        not project.pull_request_is_from_fork(pull_request)
+        )
     )
 
+
 def is_deployed(host, deployment):
     worktree_name = deployment['environment'][len(DEPLOYMENT_PREFIX):]
-    response = requests.get(
-        '{}/.git/worktrees/{}/HEAD'.format(host, worktree_name)
-    )
+    url = '{}/.git/worktrees/{}/HEAD'.format(host, worktree_name)
+    logger.info('Issuing request: GET %s', url)
```

Comment:
This feels more like debug level to me, but I don't mind leaving it info if you intended to show it by default (debug logs are not).

Reply:
I prefer to show it by default, given this is just in CI and knowing what requests fire helps any debugging.